src/index.html

```diff
@@ -5,8 +5,402 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#home" class="header__link">
+          <img 
+            src="./images/logo.png" 
+            alt="logo"
+            class="header__img" 
+          >
+        </a>
+      </div>
+
+      <div class="header__nav">
+        <nav class="nav">
+          <ul class="nav__list">
+            <a href="#" class="nav__link is-active">Apple</a>
+            <a href="#" class="nav__link">Samsung</a>
+            <a href="#" class="nav__link">Smartphones</a> 
+            <a href="#home" 
```

Comment:
Format this code according to HTML conventions (if you have many attributes - each must start from a new line)

Example:
 ```
<a 
   href="#home" 
   data-qa="nav-hover" 
   class="nav__link"
>
```